core/drand_daemon.go

```diff
@@ -171,6 +171,9 @@ func (dd *DrandDaemon) InstantiateBeaconProcess(beaconID string, store key.Store
 	dd.beaconProcesses[beaconID] = bp
 	dd.state.Unlock()
 
+	metrics.DKGStateChange(metrics.DKGNotStarted, beaconID, false)
+	metrics.ReshareStateChange(metrics.ReshareIdle, beaconID, false)
+
```

Comment:
Isn't this lacking `metrics.IsDrandNode.Set(1)` too ?

Reply:
Fixed.